printer_zpl2/README.rst

```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+ZPL II Label printing
+=====================
+
+This module extends the **Report to printer** (``base_report_to_printer``)
+module to add a ZPL II label printing feature.
+
+It's done to be used from other modules, or from python code blocks.
```

Comment:
Exactly, thanks. I'll take your statement.
I think the readme can also be improved for that, as the `_print_label` method is not mentionned.